inspirehep/modules/deposit/dojson/fields/literature.py

```diff
@@ -221,7 +221,7 @@ def languages(self, key, value):
     if value not in ('en', 'oth'):
         return [unicode(dict(languages).get(value))]
     else:
-        return [value]
+        return []
```

Comment:
Yes, that comes from the format in production. If there is a language field it is displayed close to the title. For English we don't put anything in the field so nothing is displayed.